.travis.yml

```diff
@@ -1,4 +1,14 @@
 language: go
 go: 1.7.1 
 install: true
-script: make test
+
+addons:
+  apt:
+    sources:
+    - debian-sid    # Grab shellcheck from the Debian repo (o_O)
```

Comment:
Great question. I copied it from here: https://github.com/koalaman/shellcheck/wiki/TravisCI